README/ReleaseNotes/v634/index.md

```diff
@@ -223,7 +225,18 @@ TCanvas::SaveAll({c1, c2, c3, c4}, "file.pdf");
 ```
 
 ## 2D Graphics Libraries
+* In `TGraphErrors` `TGraphAsymmErrors` and `TGraphBentErrors`, the error bars were drawn
```

Comment:
Is there some line break kicking in? Do we maybe want to have also these bullets on one single lines like all others?